scripts/windows/setup/IotEdgeSecurityDaemon.ps1

```diff
@@ -866,16 +866,21 @@ function Write-HostRed {
     Write-Host -ForegroundColor Red @args
 }
 
-function Remove-Permissions([string] $Path) {
+function Remove-BuiltinWritePermissions([string] $Path) {
     $user = 'BUILTIN\Users'
     Write-Verbose  "Remove $user permission to $Path"
     Invoke-Native "icacls ""$Path"" /inheritance:d"
     
     $acl = Get-Acl -Path $Path
-    foreach ($access in $acl.Access) { 
-        if ($access.IdentityReference.Value -eq $user)  
-        { 
-            $acl.RemoveAccessRule($access) | Out-Null
+    $write = [System.Security.AccessControl.FileSystemRights]::Write
```

Comment:
Check with Venkat that this type and `FileSystemAccessRule` below exist on IoT Core. (MSDN says they were in .Net Core 1.0, so they should be there.)

Reply:
I just tried it; it's there.